packages/ui/src/lib/utils.ts

```diff
@@ -1,9 +1,24 @@
 import { type ClassValue, clsx } from "clsx";
 import { twMerge } from "tailwind-merge";
 
+declare const process: { env: Record<string, any> };
```

Comment:
Is this currently assuming that the value will be provided by `apps/roam` 's compile script?  If we go ahead with `NEXT_API_ROOT`, maybe we should add a `.env` file to `packages/ui` and locate `NEXT_API_ROOT` there so it is decoupled.

Reply:
It is assumed, and as I understand it, `packages/ui` is not compiled individually, but as included by each app using it. So the applicable `.env` would be the app's in all cases.